src/components/Search/CompareWithBase.tsx

```diff
@@ -259,7 +253,6 @@ function CompareWithBase({
           className='form-wrapper'
           onSubmit={onFormSubmit}
           aria-label='Compare with base form'
-          reloadDocument={hasEditButton ?? true}
```

Comment:
Still a leftover from the work you did with Suspense, can you please put it back? To avoid conflicts with the other patch.